src/System.Runtime.Extensions/tests/System/MathF.netcoreapp.cs

```diff
@@ -795,6 +928,76 @@ public static void Floor(float value, float expectedResult, float allowedVarianc
+        [InlineData( 0.0f,                    float.NegativeInfinity,  3.14159265f,             float.NaN)]
+        [InlineData( 0.0f,                    float.NegativeInfinity,  float.PositiveInfinity,  float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity,  float.NegativeInfinity,  float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity, -3.14159265f,             float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity, -0.0f,                    float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity,  float.NaN,               float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity,  0.0f,                    float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity,  3.14159265f,             float.NaN)]
+        [InlineData( 0.0f,                    float.PositiveInfinity,  float.PositiveInfinity,  float.NaN)]
+        [InlineData( 1e38f,                   2.0f,                   -1e38f,                   1e38f)]
+        [InlineData( 1e38f,                   2.0f,                    float.NegativeInfinity,  float.NegativeInfinity)]
+        [InlineData( float.PositiveInfinity,  float.NegativeInfinity,  float.PositiveInfinity,  float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                    float.NegativeInfinity,  float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                   -3.14159265f,             float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                   -0.0f,                    float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                    float.NaN,               float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                    0.0f,                    float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                    3.14159265f,             float.NaN)]
+        [InlineData( float.PositiveInfinity, -0.0f,                    float.PositiveInfinity,  float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                    float.NegativeInfinity,  float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                   -3.14159265f,             float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                   -0.0f,                    float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                    float.NaN,               float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                    0.0f,                    float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                    3.14159265f,             float.NaN)]
+        [InlineData( float.PositiveInfinity,  0.0f,                    float.PositiveInfinity,  float.NaN)]
+        [InlineData( float.PositiveInfinity,  float.PositiveInfinity,  float.NegativeInfinity,  float.NaN)]
```

Comment:
Do you want to have one "happy path" case here?  Like `5, 4, 3`?

Reply:
I can always add another, but we already have `1e38f, 2.0f, -1e38f`; which tests a "happy" case and that FMA functions as intended (doing a fused operation, rather than two scalar operations).

Reply:
Fixed.